lib/openhab/dsl/items/builder.rb

```diff
@@ -176,8 +177,13 @@ class ItemBuilder
         attr_accessor :channels
         # @return [Core::Items::Metadata::NamespaceHash]
         attr_reader :metadata
+        #
         # Initial state
-        # @return [Core::Types::State]
+        #
+        # If a Proc is given, {Rules::Terse.calculated_item calculated_item}
+        # will be used to keep the item's state up to date.
```

Comment:
An example demonstrating how to use of this would be great. It could be a part of the main Builder module's description above, I just can't add a comment there.